components/write_to_file/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM --platform=linux/amd64 python:3.8-slim as base
```

Comment:
We might want to start using newer python versions since 3.8 will reach `end-of-life` before you know it. (https://devguide.python.org/versions/)